app/forms/schools/add_participant_form.rb

```diff
@@ -358,5 +358,9 @@ def appropriate_body_selected
     rescue ActiveRecord::RecordNotFound
       nil
     end
+
+    def existing_participant_profile_same_school?
+      existing_participant_profile&.school == school_cohort.school
```

Comment:
Surely ParticipantProfile::ECF.school is legacy. 
Shouldn't we use the participant_profile's latest_induction_record's school instead ?

Reply:
Yes I agree, lets use the induction records for this :+1: